.bazelrc

```diff
@@ -0,0 +1,3 @@
+startup --windows_enable_symlinks
```

Comment:
Is it possible to build openssl without these flags enabled?

Reply:
PRs welcome to enable this :)